pihole

```diff
@@ -253,35 +253,30 @@ Options:
 analyze_ports() {
   # FTL is listening at least on at least one port when this
   # function is getting called
-  if [[ $(grep -c "IPv4" <<< "${1}") -gt 1 ]] && \
-     [[ $(grep -c "IPv6" <<< "${1}") -gt 1 ]]; then
-    echo -e "  ${TICK} DNS service is listening"
+  echo -e "  ${TICK} DNS service is listening"
+  # Check individual address family/protocol combinations
+  # For a healthy Pi-hole, they should all be up (nothing printed)
+  if grep -q "IPv4.*UDP" <<< "${1}"; then
+      echo -e "     ${TICK} UDP (IPv4)"
```

Comment:
Because whitespace is a pain to deal with, how about `printf` instead?

```suggestion
      printf "\\t%s UDP (IPv4)\\n" "${TICK}"
```

Reply:
That may need to be `%b` instead of `%s` for the tick/cross.

Reply:
I followed what is already there, but no harm in improving things. I will check after dinner